### PR TITLE
Fix odd resolution handling of I420/YV12 formats

### LINKED ISSUE #1037: vaDeriveImage()/vaGetImage() doesn't fill the last column and row in chroma plane if the dimension is odd
This is same issue of i965 driver filled in https://github.com/intel/intel-vaapi-driver/issues/516.
vaDeriveImage() in media driver has the same issue unlike i965 driver though.
I expect the same gstreamer command can reproduce this issue.
https://github.com/intel/intel-vaapi-driver/issues/516#issuecomment-672166170

### PR DESCRIPTION
**Pull request by Jister**
Fix #1037

### CODE REVIEW

**Comment by FurongZhang** [comment]
I don't have any concern for this patch. Just need to run regression test to make sure no regression.

**Comment by XinfengZhang** [comment]
LGTM @sreerenjb , how about this patch?

File: media_driver/linux/common/ddi/media_libva_util.cpp [hunk 1]
```diff
@@ -208,6 +208,8 @@ VAStatus DdiMediaUtil_AllocateSurface(
                 !(mediaSurface->surfaceUsageHint & VA_SURFACE_ATTRIB_USAGE_HINT_VPP_WRITE))
             {
                 tileformat = I915_TILING_NONE;
+                alignedWidth = MOS_ALIGN_CEIL(width, 2);
+                alignedHeight = MOS_ALIGN_CEIL(width, 2);
```
Comment (uartie) [inline media_driver/linux/common/ddi/media_libva_util.cpp:212]: should this be `MOS_ALIGN_CEIL(height, 2)`?

**Comment by sreerenjb** [comment]
@XinfengZhang 
As @uartie mentioned, as long as it is not causing #1071 and #1072 , I'm okay with the patch.

File: media_driver/linux/common/ddi/media_libva_util.cpp (hunk 1 above)
Comment (sreerenjb) [reply media_driver/linux/common/ddi/media_libva_util.cpp:212]: It should be :)

**Comment by Jister** [comment]
Verified locally that the case show no corruption with latest patch. Sorry for the typo in previous patch.

**Pull request merged**